app/assets/stylesheets/components/_split-tabs.scss

```diff
@@ -45,3 +45,9 @@
     background-color: $mit-red;
   }
 }
+
+.emphasized-tab-digit-small {
```

Comment:
If this is a modified of `.split-tab-digit`, I would nest it inside `.split-tab-digit` to show that it needs to be coupled with it to work.